stripe.go

```diff
@@ -244,6 +244,7 @@ func (s *BackendConfiguration) Do(req *http.Request, v interface{}) error {
 				Type:           ErrorType(root["type"].(string)),
 				Msg:            root["message"].(string),
 				HTTPStatusCode: res.StatusCode,
+				RequestID:      res.Header.Get("Request-Id"),
```

Comment:
Yes. Even in the case that it isn't, this will just return an empty string. Since we serialize the error to JSON when printing, and we have `omitempty` in the json tag, an empty value won't show.